sites/headers-includes/sitesearch.html

```diff
@@ -5,7 +5,11 @@ <h2>{{ i18nText-search }}</h2>
 		<div class="form-group wb-srch-qry">
 			<label for="wb-srch-q" class="wb-inv">{{ i18nText-searchSite }}</label>
 			<input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" type="search" value="" size="34" maxlength="170" placeholder="{{ i18nText-searchSite }}" />
-			<datalist id="wb-srch-q-ac"></datalist>
+			<datalist id="wb-srch-q-ac">
+				data-wb-limit="10"
+				data-wb-filter-type="startWith"
+				data-wb-suggest="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/sites/search/json/canada-{{ i18nText-lang }}.json#/suggestions">
```

Comment:
Don't need to specify the domain "wet-boew.github.io".

Reply:
Changed to data-wb-suggest="https://www.canada.ca/content/dam/canada/srch-rchr/canada-{{ i18nText-lang }}.json#/suggestions